src/main/kotlin/uk/gov/gdx/datashare/models/TestEvent.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.gdx.datashare.models
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Test event")
```

Comment:
If we don't have a schema here does it not appear in the swagger? I'm not sure we need to publish this as part of our public API (and can therefore change it without it being a breaking change)

Reply:
Happy to remove